bindings/go/evmc/loader.c

```diff
@@ -0,0 +1,6 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+#include "../../../lib/loader/loader.c"
```

Comment:
Why not make this file a symlink instead? Is that not allowed by govendor or package managers?

Reply:
Do symlinks work on Windows?

Reply:
Git allows symlinks, I am sure it resolves it somehow on Windows. @gumb0 do you know by any chance what git is doing in that case?